addons/io/org.openhab.io.neeo/README.md

```diff
@@ -0,0 +1,142 @@
+
+Any device type that is not marked as "(Exclude)" will be visible to the NEEO App when searching for devices.
+
+#### Channels to Capabilities
+
+The second step is to map openHAB channels to one or more NEEO capabilities.  A NEEO capability can either be a virtual item on the screen or a hard button on the remote.
+
+For each channel, you may:
+
+1.  Press the ADD icon to add a new mapping from the openHAB channel (or DELETE icon to delete the mapping).
+2.  Specify or choose the NEEO capability type for the openHAB channel
+3.  Specify the NEEO label (or hard button) for the mapping
+4.  Optionally set the format or command for the mapping
+
+At the time of this writing, the following NEEO capability types are supported:
+
+1.  Text Label - this will simply take the toString() of the channel value and optionally format via the Java String Format found in the "Format/Command" field before sending it to the NEEO brain.
+2.  Button - this will create virtual button with the text from the "NEEO Label".  Upon pressing the button, the specified command in the "Format/Command" will be sent to the channel (or ON will be sent if nothing has been specified).  Please note that you can also specify a hard button in the "NEEO Label" - in which case nothing will appear on the NEEO remote screen and the action will occur from the NEEO remote hard button.  You must specify all the hard buttons for a capability (as specified in the NEEO SDK documentation) for the button to work.  Example: if you only defined VOLUME DOWN but not VOLUME UP - the button will not work on the remote.  Likewise, which hard buttons are active or not additionally depends on the NEEO device type.
+3.  Switch - this will create a virtual switch with the text from the "NEEO Label" and will send a ON or OFF command to the associated channel.  Additionally, a switch can be bound to hard button pairs (the VOLUME keys, the POWER ON/OFF, the CHANNELS, etc).  The command that is sent is dependent on the KEYS chosen (POWER ON/OFF will send ON/OFF to the underlying channel, all others will send an INCREASE/DECREASE). Similar to the "Button" type - please review the NEEO SDK documentation.    
+4.  Slider - this will create a virtual slider that will send the associated value to the channel.  The value sent will always be between 0 and 100.  
+5.  ImageURL - this will create an image on the remote from the toString() of the channel value (assuming it's a URL to an image).
+
+Please note the value for each of the hard buttons is specified in the NEEO SDK documentation.
+ 
+The following chart shows what openHAB command types are supported for each NEEO Capability type
+
+| NEEO Capability Type | openHAB Command Type                                                                                                         |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Great link - I've always done this manually.  However - I'm not sure what you are seeing because the table is exactly the same as I have in the document.

I think I may see your problem however - if I resize eclipse to smaller size, eclipse view causes the issue you are seening (see how line 84 and 85 in your image is taller than line 83).  I have very wide screens (36" widescreens) and the table shows perfectly.  The issue is eclipse will wrap those lines rather than provide a scrollbar.  Really not much I can do about that.